src/migration/1647526647810-init.ts

```diff
@@ -15,26 +15,30 @@ export class init1647526647810 implements MigrationInterface {
         await queryRunner.query(`ALTER TABLE "iasql_dependencies" ADD CONSTRAINT "FK_9732df6d7dff34b6f6a1732033b" FOREIGN KEY ("module") REFERENCES "iasql_module"("name") ON DELETE CASCADE ON UPDATE CASCADE`);
         await queryRunner.query(`ALTER TABLE "iasql_dependencies" ADD CONSTRAINT "FK_7dbdaef2c45fdd0d1d82cc9568c" FOREIGN KEY ("dependency") REFERENCES "iasql_module"("name") ON DELETE CASCADE ON UPDATE CASCADE`);
         await queryRunner.query(`
-            create or replace procedure until_iasql_operation(_optype iasql_operation_optype_enum, _params text[])
+            create or replace function until_iasql_operation(_optype iasql_operation_optype_enum, _params text[]) returns void
             language plpgsql
             as $$
             declare
                 _opid uuid;
                 _counter integer := 0;
                 _output text;
                 _err text;
+                _dblink_sql text;
             begin
                 select md5(random()::text || clock_timestamp()::text)::uuid into _opid;
-                insert into iasql_operation (opid, optype, params)
-                values (_opid, _optype, _params);
                 -- schedule job
-                perform graphile_worker.add_job('operation', json_build_object('opid', _opid, 'optype', _optype, 'params', _params));
-                -- commit transaction so the worker can see the row
-                commit;
+                raise notice '% started', _opid;
+                PERFORM dblink_connect('iasqlopconn', 'loopback_dblink');
+                -- _dblink_sql := format('perform graphile_worker.add_job(%L);', 'operation');
+                _dblink_sql := format('perform graphile_worker.add_job(%L, json_build_object(%L, %L, %L, %L));', 'operation', 'opid', _opid, 'optype', _optype);
+                -- raise exception '%', _dblink_sql;
+                PERFORM dblink_exec('iasqlopconn', _dblink_sql);
+                PERFORM dblink_disconnect('iasqlopconn');
+                raise notice '% scheduled', _opid;
                 -- times out after 45 minutes = 60 * 45 = 2700 seconds
                 -- currently the longest is RDS where the unit test has a timeout of 16m
-                while _counter < 2700 loop
-                    if (select end_date from iasql_operation where opid = _opid) is not null then
+                while _counter < 25 loop
```

Comment:
this is to easily test without killing the process when it doesn't poll correctly